components/VerticalTabs/MemberList.tsx

```diff
@@ -0,0 +1,33 @@
+// import { useInView } from "react-intersection-observer";
+import MemberCard, { MemberCardProps } from "../MemberCard";
+
+interface MemberListProps {
+    data: MemberCardProps[]
+}
+
+const MemberList = ({data}: MemberListProps) => {
+    return (
+        <div className="flex flex-col">
+            <div className="flex flex-row p-4">
+                <h1 className="text-green-300 text-4xl pl-4">80+ </h1>
+                <h1 className="text-white pt-2 px-4 text-xl"> Members</h1>
+                <h1 className="text-green-300 pl-6 text-4xl">800+ </h1>
+                <h1 className="text-white pt-2 px-4 text-xl"> Alumni</h1>
+            </div>
+            {/* <div className="flex space-x-4"> */}
+            <div className="grid gap-1 grid-cols-4">
```

Comment:
- Limit the max size of the grid (We possibly will have 50+ cards!)
- Increase gap
- Enable scrolling on the component
- Responsive grid layout
```jsx
 <div className="grid max-h-[80vh] grid-cols-3 gap-3 overflow-y-scroll px-2 md:grid-cols-4 lg:grid-cols-3 xl:grid-cols-4">
```